estate/models/estate_property_type.py

```diff
@@ -0,0 +1,22 @@
+from odoo import api, fields, models, _
```

Comment:
```suggestion
from odoo import  _, api, fields, models
```
ASCII value for `_` is 95 and `a` is 97.

Also the import is not used. Please find the use of `_` and why we used it?

Reply:
_ is used for Translate the term using current environment's language
[Code](https://github.com/odoo/odoo/blob/18.0/odoo/api.py#L770)